Documentation/devel/stage1-implementors-guide.md

```diff
@@ -60,6 +60,7 @@ Stage 1 must write the host PIDs of the pod's process #1 and that process's pare
 * `--net[=$NET1,$NET2,...]` to configure the creation of a contained network.
   See the [rkt networking documentation](../networking.md) for details.
 * `--mds-token=$TOKEN` passes the auth token to the apps via `AC_METADATA_URL` env var
+* `--hostname=$HOSTNAME` configures the host name of the pod. If empty, it will be "rkt-$PODUUID". Only available from ABI version 2.
```

Comment:
Probably would be easier to read if there was a separate section like "flags added in interface version 2".